File: Medium/2026-07-23-3513-Number-of-Unique-XOR-Triplets-I/solution.py
class Solution:
    def uniqueTriplets(self, nums: List[int]) -> int:
        unique_xor_values = set()
        n = len(nums)
        
        for i in range(n):
            for j in range(i, n):
                for k in range(j, n):
                    xor_value = nums[i] ^ nums[j] ^ nums[k]
                    unique_xor_values.add(xor_value)
        
        return len(unique_xor_values)

File: Medium/2026-07-23-3513-Number-of-Unique-XOR-Triplets-I/solution.java
import java.util.HashSet;

public class Solution {
    public int uniqueTriplets(int[] nums) {
        HashSet<Integer> uniqueXorValues = new HashSet<>();
        int n = nums.length;

        for (int i = 0; i < n; i++) {
            for (int j = i; j < n; j++) {
                for (int k = j; k < n; k++) {
                    int xorValue = nums[i] ^ nums[j] ^ nums[k];
                    uniqueXorValues.add(xorValue);
                }
            }
        }

        return uniqueXorValues.size();
    }
}